### PR TITLE
fix: Use v1beta1 in base VirtualMachineInstancetype definition

### LINKED ISSUE #56: Some resources still using `instancetype.kubevirt.io/v1alpha2`
/kind bug

**What happened**:

Missed by #44 

```
$ git grep -l instancetype.kubevirt.io/v1alpha2
common-instancetypes-all-bundle.yaml
common-instancetypes-bundle.yaml
common-instancetypes/instancetypes/base/VirtualMachineInstancetype.yaml
```

### PR DESCRIPTION
**Pull request by lyarwood**
**What this PR does / why we need it**:

```
$ git grep -l instancetype.kubevirt.io/v1alpha2
common-instancetypes-all-bundle.yaml
common-instancetypes-bundle.yaml
common-instancetypes/instancetypes/base/VirtualMachineInstancetype.yaml
```

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #56 

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: scripts/check.sh [hunk 1]
```diff
@@ -1,5 +1,12 @@
 #!/bin/bash
 
+# Basic instancetype.kubevirt.io version check
+INSTANCETYPE_API_VERSION="v1beta1"
+if grep -ri instancetype.kubevirt.io/v1 common-instancetypes | grep -v ${INSTANCETYPE_API_VERSION}; then
```
Comment (0xFelix) [inline scripts/check.sh:5]: Might be enough to grep just for `instancetype.kubevirt.io/`?

Comment (lyarwood) [reply scripts/check.sh:5]: That picks up various labels:

```
$ git grep instancetype.kubevirt.io/
data/common-instancetypes-bundle/common-clusterinstancetypes-bundle.yaml:apiVersion: instancetype.kubevirt.io/v1beta1
data/common-instancetypes-bundle/common-clusterinstancetypes-bundle.yaml:    instancetype.kubevirt.io/class: Compute Exclusive
data/common-instancetypes-bundle/common-clusterinstancetypes-bundle.yaml:    instancetype.kubevirt.io/description: |-
data/common-instancetypes-bundle/common-clusterinstancetypes-bundle.yaml:    instancetype.kubevirt.io/version: "1"
data/common-instancetypes-bundle/common-clusterinstancetypes-bundle.yaml:    instancetype.kubevirt.io/cpu: "8"
data/common-instancetypes-bundle/common-clusterinstancetypes-bundle.yaml:    instancetype.kubevirt.io/dedicatedCPUPlacement: "true"
data/common-instancetypes-bundle/common-clusterinstancetypes-bundle.yaml:    instancetype.kubevirt.io/hugepages: "true"
data/common-instancetypes-bundle/common-clusterinstancetypes-bundle.yaml:    instancetype.kubevirt.io/isolateEmulatorThread: "true"
data/common-instancetypes-bundle/common-clusterinstancetypes-bundle.yaml:    instancetype.kubevirt.io/memory: 16Gi
data/common-instancetypes-bundle/common-clusterinstancetypes-bundle.yaml:    instancetype.kubevirt.io/numa: "true"
data/common-instancetypes-bundle/common-clusterinstancetypes-bundle.yaml:    instancetype.kubevirt.io/vendor: kubevirt.io
```

Comment (0xFelix) [reply scripts/check.sh:5]: `apiVersion: instancetype.kubevirt.io/`?

Comment (lyarwood) [reply scripts/check.sh]: I've gone with `grep -riP "instancetype.kubevirt.io/v\d+" common-instancetypes`.

**Review by 0xFelix — APPROVED** [review]
Thanks!

/approve
/lgtm

**Pull request merged**